mobile/native/src/db/models.rs

```diff
@@ -1139,6 +1141,7 @@ impl From<Channel> for ln_dlc_node::channel::Channel {
             channel_id: value
                 .channel_id
                 .map(|cid| ChannelId::from_hex(&cid).expect("valid channel id")),
+            fake_scid: None,
```

Comment:
🤔 I think we do have this value in `value`.

Reply:
This is on the app, and on the app we don't have it; the model was not extended to reflect this value.

Reply:
Ah, good point. Not ideal to have a shared model, but we can clean that up at some point in the future.